### DISCUSSION TITLE
Scroll to the desired id

### DISCUSSION THREAD

**Discussion by AleksandrDomashenko** [swup/swup#627] [Help] [answered]
Using Swup, I go to another page, there are no problems with this. But it is not possible to scroll to the desired element with a certain id. Please tell me how to implement it.
ScrollPlugin is connected, scroll happens all the way to the top of the page.

**Comment by hirasso** [comment, marked as answer]
Hi @OleksandrDomashenko, could you create a reduced test case using our [starter template on CodeSandbox](https://codesandbox.io/s/swup-test-case-template-6ci5si)? My best guess is that you have a typo in your link.

**Reply by AleksandrDomashenko** [reply]
I solved the above problem as follows.

The script is on the main page of the site:
```
<script>
      window._swup = new Swup({
        plugins: [new SwupScrollPlugin()]
      });
</script>
```
In the link for the animated transition, I added the necessary ID.

`href="/pages/electrical_safety/safety.html#safety"`

Thanks for the quick response.
Maybe it will be useful to someone.
